### ISSUE TITLE
My ip blocked by youtube

### ISSUE THREAD

**Issue by SaadAmien20** [yt-dlp/yt-dlp#9890]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

I am working on a big project  and I want to download more than 2 million video, but after I downloaded around 124K video, I got Message, `Your IP is blocked By Youtube `
I expected that the YouTube site determine specific number, we should not exceed it. if that right.

Can I know what that number which I should not exceed per day or per week.

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

**Comment by m-attz** [comment]
As far as I know YT-DLP doesn't use the official Google/YouTube API so finding a specific limit would be hard. If your IP was blocked around the 124,000 mark then it most likely detected that your IP was "abusing" due to the high amount of requests that were not coming from the official API.

You could take a guess by looking at the API quota limits here:
[https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Understanding-YouTube-API-Quota-Limits](url)


You could attempt to circumvent this by using proxies.

**Comment by heyeanne34** [comment]
hi what do u need 2 million video for? @SaadAmien20

**Comment by bashonly** [comment]
The exact threshold and/or trigger(s) for the ban are not known and this question is out-of-scope for this tracker.

**Comment by KyrbySwiUtch** [comment]
I've had my IP blocked today and I havn't even downloaded anything today. In the past I rarely downloaded much, so I feel like Youtube just bans IP's for other, currently unknown reasons. Does anyone have any idea what the causes could be, since in my case I really didn't do much of anything and got banned. Also, is there a way to tell if IP bans are permanent?
